package.json

```diff
@@ -29,6 +29,8 @@
     "@ant-design/icons": "4.6.2",
     "ahooks": "2.10.0",
     "antd": "4.15.0",
+    "axios": "0.21.1",
+    "axios-mock-adapter": "1.19.0",
```

Comment:
Hey @634750802 , as we discussed last Friday, I created an independent mock API server in `packages.mockApi` folder, please refer to my last PR at https://github.com/pingcap/tug-website/pull/62

However, apart from mock APIs, your PR also includes a lot of stuff, like UI components and dropdown selections. So I think I should let you decide how to update your PR.